cmd/kubermatic-installer/cmd_deploy.go

```diff
@@ -139,14 +144,25 @@ func DeployAction(logger *logrus.Logger, versions kubermaticversion.Versions) cl
 				deployHelmBinaryFlag.EnvVar)
 		}
 
-		logger.WithFields(fields).Info("🛫 Initializing installer…")
+		var kubermaticStack stack.Stack
+		stackName := ctx.Args().First()
+		switch stackName {
+		case "kubermatic-seed":
+			kubermaticStack = kubermaticseed.NewStack()
+
+		case "kubermatic-master":
+			fallthrough
 
-		supportedProviders := kubermatic.SupportedStorageClassProviders()
-		chosenProvider := ctx.String(deployStorageClassFlag.Name)
-		if chosenProvider != "" && !supportedProviders.Has(chosenProvider) {
-			return fmt.Errorf("invalid storage class provider %q given (--%s)", chosenProvider, deployStorageClassFlag.Name)
+		default:
+			kubermaticStack = kubermaticmaster.NewStack()
 		}
```

Comment:
What is a Kubermatic Stack supposed to represent? I'm not sure I understand this, but I have a few questions:

* What if a user provides some random string or doesn't provide any string?
* Is it expected that we always default to the `kubermatic-master` stack?

Reply:
A "stack" in installer terms is "a combination of Helm charts meant to provide a common functionatlity". We have

* KKP master
* KKP seed
* monitoring
* logging
* (backup maybe)

stacks, as I see them currently.

W.r.t. to the other questions: That switch statement was a complete brainfart. I fixed it now, the original intent was to just make `./installer deploy` default to `./installer deploy kubermatic-master`.

Reply:
Okay, that makes sense, thank you for explaining it to me! The switch statement looks okay now.